ospsuite-r-specifics/collaboration_guide.md

```diff
@@ -0,0 +1,122 @@
+# R Development Collaboration Guide
+
+## Rules
+
+The following set of rules should be followed by all contributors and checked by
+all reviewers of the OSPS R projects.
+
+1. Each change made to the codebase should have a clear purpose and exist in an
+isolated context.  
+  1.1 Each change should be related to an issue created on the project's repository.  
+  1.2 Each change should be made in a separate branch.  
+  1.3 Each change should be proposed through a pull request.  
+  
+2. Each change or addition to the code should be documented and controlled.  
+  2.1 Each change in the code should be reflected in the documentation. If 
+  documentation is not present, then it should be created.  
+  2.2 Each change should be tested. If new cases emerge, then they should be 
+  tested, even if some tests are already present. If no tests are present, then
+  they should be created.  
+  
+3. Each change should be easily reviewable, understandable and accessible.   
+  3.1 Each change should have a corresponding entry in the `NEWS` file.  
```

Comment:
Some changes do not necessarily require such an entry, e.g. adding new tests for existing functionality etc.

Reply:
True, same comment as above